packages/web/src/scss/components/RadioField/README.md

```diff
@@ -0,0 +1,22 @@
+# RadioField
+
+Use RadioField when you have a group of mutually exclusive choices and only one selection from the group is allowed. It has input and label. It can be disabled or have an error state. The label can be hidden.
```

Comment:
I know it's not in the design either but want to ask anyway: don't we need validation states for this component?

Reply:
Well, I did it according to how the checkbox was made. Don't we want to keep it consistent?

Reply:
Correct, we may get back to it later. 👍🏻